src/Discretization/FirstOrderZonotopeModule.jl

```diff
@@ -103,7 +104,9 @@ function discretize(ivp::IVP{<:CLCCS,<:AbstractZonotope}, δ, alg::FirstOrderZon
     # compute bloating factor β
     U = next_set(inputset(ivp), 1)
     μ = norm(U, Inf)
-    β = ((exp(norm_A * δ) - 1) * μ) / norm_A
```

Comment:
Please do not add two empty lines (at most one).

I think the `norm_A ==` part should be removed.

Any reason to rewrite the "else" case? I suggest to keep the old expression.